__tests__/app.test.js

```diff
@@ -108,3 +113,115 @@ describe("app", () => {
+        favorite_food: "Pizza",
+      };
+      return request(app)
+        .post("/api/reviews/1/comments")
+        .send(firstComment)
+        .expect(201)
+        .then(({body})=> {
+          expect(body.comment.body).toBe(firstComment.body);
+    });
+  });
+
+    test("400: responds with error message when username not found", () => {
+      const secondComment = {
+        username: "nonexistinguser",
+        body: "This is an amazing game, unfortunately no spider characters though!",
+      };
+      return request(app)
+        .post("/api/reviews/1/comments")
+        .send(secondComment)
+        .expect(404)
+        .then(({body})=> {
+          expect(body.msg).toBe("Route Not Found")
+    });
+    });
+
+    test("400: responds with error message when review_id is out of range", () => {
+      const thirdComment = {
+        username: "bainesface",
+        body: "This is an amazing game, unfortunately no spider characters though!",
+      };
+      return request(app)
+        .post("/api/reviews/999/comments")
+        .send(thirdComment)
+        .expect(404)
+        .then(({body}) => {
+          expect(body.msg).toBe("Route Not Found");
+        })
+    });
+
+    test("400: responds with error message when review_id is not a number", () => {
+      const fourthComment = {
+        username: "bainesface",
+        body: "This is an amazing game, unfortunately no spider characters though!",
+      };
+      return request(app)
+        .post("/api/reviews/abc/comments")
+        .send(fourthComment)
+        .expect(400)
+        .then(({body}) => {
+          expect(body.msg).toBe("Bad Request");
+        })
+    });
+
+    test("400: responds with error message when required fields are not filled in (username)", () => {
+      const fifthComment = {
+        username: "",
+        body: "This is an amazing game, unfortunately no spider characters though!",
+      };
+      return request(app)
+        .post("/api/reviews/1/comments")
+        .send(fifthComment)
+        .expect(400)
+        .then(({body}) => {
+          expect(body.msg).toBe("Required fields are not filled in");
+        })
+    });
+
+    test("400: responds with error message when required fields are not filled in (body)", () => {
+      const sixthComment = {
+        username: "bainesface",
+        body: "",
+      };
+      return request(app)
+        .post("/api/reviews/1/comments")
+        .send(sixthComment)
+        .expect(400)
+        .then(({body}) => {
+          expect(body.msg).toBe("Required fields are not filled in");
+        })
+    });
+    test("400: responds with error message when comment is empty", () => {
+      return request(app)
+        .post("/api/reviews/1/comments")
+        .send()
+        .expect(400)
+        .then(({body}) => {
+          expect(body.msg).toBe("Required fields are not filled in");
+        })
+    });
+})
+});
```

Comment:
I would like you to extend your testing for the post comment response. You are currently only checking the username and the body, please check for all the other expected fields